.svglintrc.mjs

```diff
@@ -440,17 +433,21 @@ export default {
           upperCurveCommand,
           upperShorthandCurveCommand,
         ];
-        const curveCommands = [...lowerCurveCommands, ...upperCurveCommands];
-        const commands = [
-          ...lowerMovementCommands,
-          ...lowerDirectionCommands,
-          ...upperMovementCommands,
-          ...upperDirectionCommands,
-          ...curveCommands,
-        ];
+        const curveCommands = lowerCurveCommands.concat(upperCurveCommands);
+        const commands = [].concat.apply(
+          [],
+          [
+            lowerMovementCommands,
+            lowerDirectionCommands,
+            upperMovementCommands,
+            upperDirectionCommands,
+            curveCommands,
+          ],
+        );
```

Comment:
```suggestion
        const curveCommands = [...lowerCurveCommands, ...upper];
        const commands = [
          ...lowerMovementCommands,
          ...lowerDirectionCommands,
          ...upperMovementCommands,
          ...upperDirectionCommands,
          ...curveCommands,
        ];
```

Enforces the use of [the spread operator (`...`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) over outdated patterns. This also helps keep consistency by using a flexible operator instead of:

- `Array.from(...)`
- `Array#concat(...)`
- `Array#slice()`
- `Array#toSpliced()`
- `String#split('')`